tests/test_parallel_upload.cpp

```diff
@@ -169,7 +168,7 @@ static int gr_teardown(void **unused)
 int main(void) {
   const struct CMUnitTest tests[] = {
     cmocka_unit_test_teardown(test_small_file_concurrent_upload, teardown),
-    cmocka_unit_test_teardown(test_large_file_multipart_upload, teardown),
+    //cmocka_unit_test_teardown(test_large_file_multipart_upload, teardown),
```

Comment:
ok, this is the main change of this PR, right?

Reply:
Yes.